marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/flow.sjs

```diff
@@ -139,6 +139,12 @@ class Flow {
     let uris = null;
     if (options.uris) {
       uris = this.datahub.hubUtils.normalizeToArray(options.uris);
+
+      if (true === options.excludeAlreadyProcessed || "true" === options.excludeAlreadyProcessed) {
```

Comment:
Another option here is `options.excludeAlreadyProcessed && xs.boolean(options.excludeAlreadyProcessed)`

Reply:
Thanks, I'll make that change. I initially only had the boolean true value there, but felt like supporting "true" was worthwhile in case a user accidentally puts in "true" in their options - seems expected that that would still work.

Reply:
Made the change, and also added a new trace event so that the impact of this feature can be logged.